composer.json

```diff
@@ -26,11 +26,15 @@
     "require-dev": {
         "amphp/phpunit-util": "^1.0",
         "friendsofphp/php-cs-fixer": "^2.9",
-        "phpunit/phpunit": "^6.5"
+        "phpunit/phpunit": "^6.5",
+        "php-coveralls/php-coveralls": "^1.0"
```

Comment:
We use the `.phar` by design, because a developer never needs the tool locally and it might avoid possible future conflicts.